src/VideoStreaming/VideoReceiver.cc

```diff
@@ -395,7 +395,7 @@ VideoReceiver::_makeSource(const QString& uri)
         gst_element_foreach_src_pad(source, _padProbe, &probeRes);
 
         if (probeRes & 1) {
-            if (probeRes & 2) {
+            if (probeRes & 2 && !_videoSettings->lowLatencyMode()->rawValue().toBool()) {
```

Comment:
There is also another place (https://github.com/mavlink/qgroundcontrol/blob/Stable_V4.0/src/VideoStreaming/VideoReceiver.cc#L238) where rtpjitterbuffer could be inserted. I think lowLatencyMode() option should be consistent across all cases - please add it.